intelmq/lib/utils.py

```diff
@@ -860,13 +862,22 @@ def list_all_bots() -> dict:
     from intelmq.lib.bot import Bot  # noqa: prevents circular import
     bot_parameters = dir(Bot)
 
-    base_path = resource_filename('intelmq', 'bots')
+    ns_pkg = intelmq.bots
 
-    botfiles = [botfile for botfile in pathlib.Path(base_path).glob('**/*.py') if botfile.is_file() and botfile.name != '__init__.py']
-    for file in botfiles:
-        file = Path(file.as_posix().replace(base_path, 'intelmq/bots'))
+    def iter_ns(pkg):
+        botmodules = []
+        for path, name, is_pkg in iter_modules(pkg.__path__, pkg.__name__ + '.'):
+            if is_pkg:
+                botmodules = botmodules + iter_ns(importlib.import_module(name))
```

Comment:
This line causes the current unit test failures (import of a SyntaxError-throwing-bot fails with a SyntaxError, which is not handled correctly here). An example of catching the exception is nine lines below.